org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/commons/TextDocumentTest.java

```diff
@@ -155,6 +156,18 @@ public void testEmptyDocumentInc() throws BadLocationException {
 
 	}
 
+	@Test
+	public void testGetLineInformation() throws BadLocationException {
+		Assertions.assertThrows(BadLocationException.class,
+				() -> {
+					TextDocument document = new TextDocument("", "");
+					document.setIncremental(true);
+					Position position = document.positionAt(0);

```

Comment:
Also just hit me that `positionAt` also throws `BadLocationException`, and if that did happen, this would just pass. I would just remove this line and have `Position position = new Position(-1, 0);`